modules/manage/partials/mountable-topics.adoc

```diff
@@ -0,0 +1,172 @@
+    }, 
+    {
+      "topic": "<topic-2-name>"
+    }, 
+    {
+      "topic": "<topic-3-name>"
+    }
+  ]
+}
+```
+
+You may optionally include the topic namespace (`ns`). Default value: `kafka`
+--
+======
+
+
+== Mount a topic to a cluster
+
+[tabs]
+======
+rpk::
++
+--
+In your target cluster, run this command to mount a topic from object storage:
+
+```
+rpk topic mount <source-topic-name>
+```
+
+You can also rename the topic as you mount it to the target cluster:
+
+```
+rpk topic mount <namespace>/<source-topic-name> --to <namespace>/<new-topic-name>
+```
+--
+Admin API::
++
+--
+To mount topics to a target cluster using the Admin API, make a POST request to the `/v1/topics/mount` endpoint. Specify the names of the topics in the request body:
+
+```
+curl -X POST http://localhost:9644/v1/topics/mount -d {
+  "topics": [
+    {
+      "source_topic": {"ns": "kafka", "topic": "<source-topic-1-name>"}, 
+      "alias": {"ns": "kafka", "topic": "<new-topic-1-name>"}
+    }, 
+    {
+      "source_topic": {"ns": "kafka", "topic": "<source-topic-2-name>"}
+    }, 
+    {
+      "source_topic": {"ns": "kafka", "topic": "source-topic-3-name"}, 
+      "alias": {"ns": "kafka", "topic": "<new-topic-3-name>"}
+    }
+  ]
+}
+```
+
+* `ns` is the topic namespace. This field is optional. Default value: `kafka`
+* To rename a topic in the target cluster, use the optional `alias` object in the request body. In the example, topics 1 and 3 are given new names in the target cluster, while topic 2 retains its original name.
+
+--
+
+======
+
+When the migration is complete, the target cluster handles produce and consume workloads for the topics.
+
+== Monitor progress
+
+Issue a GET request to the `/migrations` endpoint to view the state of topic migrations:
+
+```
+curl http://localhost:9644/v1/migrations 
+```
+
+The response returns the state of existing migrations:
+
+|===
+| Migration state | Outbound migration (unmount) | Inbound migration (mount)
+
+| `planned`
+| Redpanda has initialized a migration.
+|
+
+| `preparing`
+| Redpanda copies all possible topic data including topic manifests to the destination bucket or container in object storage.
+| Redpanda recreates the topics in a disabled state in the target cluster. The cluster allocates partitions but does not add log segments yet. Topic metadata is populated from the topic manifests found in object storage.
+
+| `prepared` 
+| The migration is ready to execute. In this state, the cluster still accepts client reads and writes for the topics.
+| 
+
+| `executing` 
+| The cluster rejects client reads and writes for the topics. Redpanda uploads any remaining topic data that has not yet been copied to object storage.
+| The target cluster starts topic partitions with log segments downloaded from object storage.
+
+| `executed` 
+| All unmounted topic data from the cluster is available in object storage.
+| All mounted topic data is available in the target cluster.
```

Comment:
Not yet. No big milestone to boast about here.

Reply:
> Not yet. No big milestone to boast about here.

I can leave this empty for the`Inbound migration` column but it would be nice to provide a short description

Reply:
Each -ed state is only an indicator of the work in the correspondent -ing state completed. This applies to `cut_over`/`finished` pair as well. See https://github.com/redpanda-data/docs/pull/725#discussion_r1795571115 for what is done in `executing`. Stages are there not necessarily to do a lot of work in each of them, but rather to replicate the results between them in order to guard from race conditions in case cluster nodes or network between them goes down.